### PR TITLE
Add default adapter for CLI and handle an error when it is invalid

### PR DESCRIPTION
**Pull request by AlexZhovnuvaty**
Signed-off-by: Alexander Zhovnuvaty <a.zhovnuvaty@gmail.com>

### CODE REVIEW

**Comment by AlexZhovnuvaty** [comment]
Observed the following error in this pull request: "continuous-integration/jenkins/pr-head — This commit cannot be built". Do not know what is the reason and how to deal with it? Please assist.

**Comment by rberg2** [comment]
Hello, that was caused by a "white list" check on our build systems. I have restarted this build.

**Comment by AlexZhovnuvaty** [comment]
> Could you split this into two commits? The changes are quite distinct, and then a more detailed commit message can be made for both.

I have split it into two. Pls check if current state is Ok. Otherwise pls provide best option to address this comment.

File: cli/sawtooth_cli/main.py [hunk 1]
```diff
@@ -184,3 +184,7 @@ def main_wrapper():
     except:
         traceback.print_exc(file=sys.stderr)
         sys.exit(1)
+
+
+if __name__ == '__main__':
```
Comment (vaporos) [inline cli/sawtooth_cli/main.py]: This is provided by the bin/sawtooth script in the repo, or by the auto-generated wrapper script when installed. This non-executable file doesn't need it.

**Comment by vaporos** [comment]
> > Could you split this into two commits? The changes are quite distinct, and then a more detailed commit message can be made for both.
> 
> I have split it into two. Pls check if current state is Ok. Otherwise pls provide best option to address this comment.

Do a rebase instead of a merge so you only have the new commits. Something like "git rebase -i upstream/master" (you will need to use --force when you push it back up).

File: cli/sawtooth_cli/main.py (hunk 1 above)
Comment (AlexZhovnuvaty) [reply cli/sawtooth_cli/main.py]: Done. This was removed

**Comment by AlexZhovnuvaty** [comment]
> > > Could you split this into two commits? The changes are quite distinct, and then a more detailed commit message can be made for both.
> > 
> > 
> > I have split it into two. Pls check if current state is Ok. Otherwise pls provide best option to address this comment.
> 
> Do a rebase instead of a merge so you only have the new commits. Something like "git rebase -i upstream/master" (you will need to use --force when you push it back up).

Done. This branch includes 2 new commits only.

**Pull request merged**